3rdparty/CMakeLists.txt

```diff
@@ -36,6 +36,9 @@ add_subdirectory(openssl openssl EXCLUDE_FROM_ALL)
 # volk
 if (Vulkan_FOUND)
 	set(VOLK_PULL_IN_VULKAN ON)
```

Comment:
btw what does this function do?

Reply:
I repeat my question

Reply:
it makes look for vulkan sdk in the system